### ISSUE TITLE
SSL by default?

### ISSUE THREAD

**Issue by wibeasley** [nutterb/redcapAPI#48]
@nutterb or @SteveLane 
I'm reviewing & updating parts of the [Troubleshooting document](http://htmlpreview.github.io/?https://github.com/OuhscBbmc/REDCapR/blob/master/inst/doc/TroubleshootingApiCalls.html) [we wrote](https://github.com/OuhscBbmc/REDCapR/pull/60).

Does redcapAPI use SSL by default now?  If so, this section of [the document](https://github.com/OuhscBbmc/REDCapR/blob/master/vignettes/TroubleshootingApiCalls.Rmd) needs to be updated:

>This check avoids SSL in order to simplify the troubleshooting.  SSL is supported by default in the [PyCap](http://pycap.readthedocs.org/en/latest/) and [REDCapR](https://github.com/OuhscBbmc/REDCapR) packages.

**Comment by SteveLane** [comment]
Isn't that server-side? Your example in that document uses SSL, it just doesn't verify the certificate.

**Comment by wibeasley** [comment]
My fault for being unclear.  Does redapAPI use SSL *verification* by default now.

**Comment by SteveLane** [comment]
Clearly, I should have made that conclusion anyway...
The answer would be yes, given redcapAPI uses httr, and the default in cURL is to verify peer.
I've confirmed by requesting (POST/GET) information from a self-signed server, which results in the following:
```
> x <- httr::POST(url=rcon$url, body=.params, config=rcon$config)
Error in function (type, msg, asError = TRUE)  : 
  Peer's Certificate issuer is not recognized.
```

**Comment by wibeasley** [comment]
Thanks for doing that, @SteveLane.  I've updated the document.  https://github.com/OuhscBbmc/REDCapR/blob/master/vignettes/TroubleshootingApiCalls.Rmd

@nutterb and I wrote this to be as language & package agnostic as possible.  I keep meaning to request that it be added into the official code or something.  It seems to serve our original intent (eg, https://groups.google.com/d/msg/project-redcap/p-9v51TbK6o/oAWrG6zFjQYJ).  If you want to add something, or change how redcapAPI is represented in the future, feel free to submit a pull request
